docs/content/basics.md

```diff
@@ -48,43 +48,67 @@ It should also have added a default route of `example.demo.localflynn.com` point
 at the `example-web` service:
 
 ```
-ROUTE                             SERVICE      ID                                         STICKY  LEADER  PATH
-http:example.demo.localflynn.com  example-web  http/2e37467e-08fc-47e5-853b-4f0574cb6871  false   false   /
+ROUTE                          SERVICE      ID                                         STICKY  LEADER  PATH
+http:example.1.localflynn.com  example-web  http/5e47ae73-2812-4d67-8353-0a33007eb1d0  false   false   /
```

Comment:
Drop this change, the rest of the example references `demo.localflynn.com` and there is no need to change it.